server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/InheritanceAction.java

```diff
@@ -134,5 +126,13 @@ private Statistics(DbSession dbSession, OrganizationDto organization) {
       countRulesByProfileKey = dao.countActiveRulesByProfileKey(dbSession, organization);
       countOverridingRulesByProfileKey = dao.countActiveRulesForInheritanceByProfileKey(dbSession, organization, ActiveRuleDto.OVERRIDES);
     }
+
+    private long getActiveRuleCount(String profileKey) {
```

Comment:
what's the benefit of having single-statement private methods that are called once ? It adds 3 lines of code whereas I perfectly understand `countRulesByProfileKey.getOrDefault(profileKey, 0L)`.